ait/gui/__init__.py

```diff
@@ -389,8 +389,15 @@ def telem_handler(session):
         for k, v in tlm.getDefaultDict().iteritems():
             packet_dict[v.uid] = v
 
+        notif_thrshld = ait.config.get('notifications.options.threshold')
```

Comment:
Let's go ahead and set sane defaults here. Otherwise the behavior could end up being a bit off. For instance, as is the `notif_thrshld` variable ends up set as `None` if my config doesn't have a value for `notifications.options.threshold` and I enable monitoring.

You can do this with a one line for both of these values
```
notif_thrshld = ait.config.get('notifications.options.threshold', 1)
notif_freq = ait.config.get('notifications.options.frequency', float('inf'))
```